src/translations/en.ts

```diff
@@ -863,4 +864,5 @@ export const en = {
 
     ...nationalitiesNames,
     ...customTranslations,
+    ...Plugins.getTranslations('en'),
```

Comment:
Need to give ability to add new language via plugin
<img width="565" alt="Снимок экрана 2020-09-15 в 11 12 34" src="https://user-images.githubusercontent.com/25403667/93183977-5d195700-f744-11ea-9990-c679b4b03dd5.png">

Reply:
as for me it's useless because plugins translations file includes only plugin's related strings, and add a new language which will contain only plugins translations is weird

Reply:
Idea was that it can be a plugin to add new language. What do you think?